sigs/cncf-sigs.md

```diff
@@ -0,0 +1,252 @@
+
+* SIG Chairs and Tech Leads may be unassigned from the SIG at any time following a 2/3 majority vote of the TOC
+
+### Governance
+
+* All SIGs inherit and follow the CNCF TOC Operating Principles.
+
+* SIGs must have a documented governance process that encourages community participation and clear guidelines to avoid biased decision-making.  
+
+    * NOTE: aim here is to align with "minimal viable" model of the CNCF projects, and only have such governance as is needed, not anything too burdensome
+
+* They may grow a set of practices over time in the same way as an OSS Project, provided this is consistent with CNCF Operating Principles. 
+
+* As with CNCF Projects all exceptions and disputes are handled by TOC with CNCF Staff help
+
+### Budget & Resource
+
+* No formal systematic budget at this time, other than commitment of CNCF executive staff to provide named person as liaison point.
+
+* Just as CNCF Projects may have "help" offered by CNCF and may ask for things via the [ServiceDesk](https://github.com/cncf/servicedesk), the SIGs may do this.
+
+## Retirement
+
+* In the event that a SIG is unable to regularly establish quorum, or fulfill the responsibilities and/or regularly report to the TOC, the TOC will:
+
+    * Consider retiring the SIG after 3 months
+
+    * Must retire the SIG after 6 months
+
+* The TOC may, by means of a 2/3 majority vote, declare "no confidence" in the SIG.  In this event, the TOC may then vote to retire or reconstitute the SIG.
+
+## Initial SIGS
+
+To bootstrap the process, the TOC proposes the following SIGs, and projects assigned to each SIG. Clearly all of these SIG’s will not be fully-formed overnight or begin operating immediately, so the TOC itself will fulfill the duties of not-yet-formed SIG’s until they are.  We can however, fairly immediately, assign one voting member of the TOC as liason for each SIG, and prioritize the order of formation of the SIGs, starting immediately with the most pressing ones. 
+
+<table>
+  <tr>
+    <td>Name (to be finalised)</td>
+    <td>Area</td>
+    <td>Current CNCF Projects</td>
+  </tr>
+  <tr>
+    <td>Traffic</td>
+    <td>networking, service discovery, load balancing, service mesh, RPC, pubsub, etc.</td>
+    <td>Envoy, Linkerd, NATS, gRPC, CoreDNS, CNI</td>
+  </tr>
+  <tr>
+    <td>Observability</td>
+    <td>monitoring, logging, tracing, profiling, etc.
+</td>
+    <td>Prometheus, OpenTracing, Fluentd, Jaeger, Cortex, OpenMetrics, </td>
+  </tr>
+  <tr>
+    <td>Governance</td>
+    <td>security, authentication, authorization, auditing, policy enforcement, compliance, GDPR, cost management, etc</td>
+    <td>SPIFFE, SPIRE, Open Policy Agent, Notary, TUF,  Falco, </td>
+  </tr>
+  <tr>
+    <td>App Dev, Ops & Testing</td>
+    <td>PaaS, Serverless, Operators,... CI/CD,  Conformance, Chaos Eng, Scalability and Reliability measurement etc.</td>
+    <td>Helm, CloudEvents, Telepresence, Buildpacks, (CNCF CI)</td>
+  </tr>
```

Comment:
In the meeting, there are comments on separating "Ops and Testing" from "App Dev". I support this separation since they are very different.  For example, "Serverless involves very different technologies/topics from Chaos Eng".  "Ops and Testing" could span a large scope and many projects in other proposed SIGs will involve Ops and testing.  It is worthwhile to have a separate "Ops and Testing"  SIG.  "App Dev" itself could also span a large scope.

Reply:
@cathyhongzhang Yes, others have expressed similar concerns.  The problem is that we don't yet seem to have a clearly agreed-upon seam along which to split this SIG. I would propose that we start with a single SIG here, and one of their first deliverables will be to survey and document the space, decide whether to split into two SIGs, and if so, exactly where.

An alternative approach might be to start with 2 separate SIGs by name, but in reality they would need to work together very closely in the beginning to define their charters so as to avoid undesirable overlaps.   Personally I would prefer to start by calling that group one SIG that's deciding where to split.

Reply:
I think we should start with 2 separate SIGs: PaaS, Serverless etc. go to one SIG while CI/CD, Conformance, Chaos Eng, Scalability and Reliability measurement etc. go to "Ops and Testing" SIG. Mixing them into one SIG brings more confusion and inefficiency than benefit.

Reply:
+1 on splitting the sigs later.

Reply:
I agree with @cathyhongzhang, it's already 'complex' to write a clear and focused charter, and those topics are broad with few relation points.